lego/utils/functions.py

```
@@ -37,3 +43,26 @@ def insert_abakus_groups(tree, parent=None):
                 name=key, defaults={**kwargs, "parent": parent}
             )[0]
         insert_abakus_groups(value[1], node)
+
+
+def request_plausible_statistics(obj: BasisModel, url_root: Optional[str]) -> Response:
+    created_at = obj.created_at.strftime("%Y-%m-%d")
+    now = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)